### ISSUE TITLE
[Log/UI] Display log with '\n' as multiple lines in UI

### ISSUE THREAD

**Issue by EinavDanielDX** [SigNoz/signoz#2435]
Currently the UI is showing a log that has break lines (\n) in one long line. is it possible to make it display multiple lines?

example log body:
```
"[2023/03/07 14:33:35] [ info] inotify_fs_add(): inode=1537505 watch_fd=195 name=/var/log/containers/signoz-k8s-infra-otel-agent-57dls_logging_signoz-k8s-infra-otel-agent-dd4175ac2a39f9d058c056402bab1e7ec01b7f9f0b8dc3e5c68403724ff89b2f.log\n  File \"/usr/local/lib/python3.10/site-packages/flask/ctx.py\", line 350, in match_request\n    result = self.url_adapter.match(return_rule=True)\n  File \"/usr/local/lib/python3.10/site-packages/werkzeug/routing.py\", line 2041, in match\n    raise NotFound()"
```

log in ui:
![image](https://user-images.githubusercontent.com/97506919/223456574-a76b11c8-8b8b-4c48-ac3c-74d70106d075.png)

**Comment by nityanandagohain** [comment]
> It should be seen/shown as one, not multiple log lines otherwise it's not really useful

I think you are quoting this from a perspective of list view, raw data view will eliminate that.

**Comment by srikanthccv** [comment]
My point was view independent. Users would find it helpful to see the stacktrace as a whole (independent of any view selection) and they shouldn't need to parse anything at the collector.

**Comment by palashgdev** [comment]
@srikanthccv @nityanandagohain @EinavDanielDX any action item from here ?

**Comment by srikanthccv** [comment]
Not yet; I just shared what I would find ideal if I were a user.

**Comment by EinavDanielDX** [comment]
I agree, being able to read a stacktrace in one block is an essential. Otherwise it makes the logs screen not practical.

**Comment by palashgdev** [comment]
closing for staleness please open if there is any issue
